lib/ship.rb

```diff
@@ -0,0 +1,16 @@
+class Ship
+  attr_reader :name, :length, :health
```

Comment:
Can we space this like this :  
    attr_reader :name,
                        :length,
                         :health